modules/vaos/app/policies/vaos_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+VAOSPolicy = Struct.new(:user, :vaos) do
+  def access?
+    Flipper.enabled?('va_online_scheduling', user) && user.loa3?
```

Comment:
Please consider `va_online_scheduling` to `config/features.yml` with a description.  While what you have is practically the same thing, I'd prefer to see all disabled features in the admin UI.